web/src/components/atoms/DownloadButton/index.tsx

```diff
@@ -19,7 +19,16 @@ const DownloadButton: React.FC<DownloadButtonProps> = ({
   ...props
 }) => {
   const handleDownload: MouseEventHandler<HTMLElement> | undefined = () => {
-    fileDownload(url, filename);
+    fetch(url, {
```

Comment:
@nourbalaha , maybe use `async/await` while fetching API to keep the codebase consistent?

Reply:
Done!